src/control.jl

```diff
@@ -691,7 +691,7 @@ function init_b(
 
         # subtract plasma contribution
         # Note: Br = cocos.sigma_RpZ * dψ_dZ / Bp_fac / r and Bz = -cocos.sigma_RpZ * dψ_dR / Bp_fac / r
-        b[k] -= cocos.sigma_RpZ * (dψpl_dZ(r, z) * cos(cp.θ) - dψpl_dR(r, z) * sin(cp.θ)) / Bp_fac / r
+        b[k] -= cocos.sigma_RpZ * (dψpl_dZ(r, z) * cos(cp.θ) + dψpl_dR(r, z) * sin(cp.θ)) / Bp_fac / r
```

Comment:
The `Note` in the comment needs to be updated. And at first glance, I would think this is wrong since the cross product should give opposite signs on the two terms. Perhaps you want to define Br and Bz separately, then combine them into `b[k]`. 

N.B.: this comment applies here and in other functions below.